deploy/01-config.yaml

```diff
@@ -4,14 +4,25 @@ metadata:
   name: event-exporter-cfg
   namespace: monitoring
 data:
-  config.yaml: |
+  config.yaml: |-
     logLevel: error
     logFormat: json
     route:
       routes:
         - match:
-            - receiver: "dump"
+            - receiver: "k8s"
```

Comment:
Are we not filtering  events here ? We should send only relevant events .

Reply:
Removing this file.
Will add in internal repo.

Reply:
this repo we can use as  internal repo only ? Its better to add it here only